database/migrations/2020_05_08_115000_add_booking_statuses.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Tipoff\Statuses\Models\Status;
+
+class AddBookingStatuses extends Migration
+{
+    public function up()
+    {
+        Status::publishStatuses('booking', [
+            'Pending',
+            'Cancelled',
+            'Confirmed',
+        ])
```

Comment:
@pdbreen Is this the correct way to add the Booking Statuses?

Reply:
Yes - that will do it.

Reply:
Thank you. That is a slick function.